provisioning/roles/utils/redis/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Ensure Redis is installed.
+  apt:
+    name: redis-server
+    state: latest
+  become: true
+
+- name: Ensure Redis' data directory exists.
+  file:
+    path: '{{ redis_data_directory }}'
+    owner: 'redis'
+    group: 'redis'
+    mode: '0755'
+  become: true
+
+- name: Configure Redis' overrides.
+  template:
+    src: 'etc/redis/redis.conf'
+    dest: '/etc/redis/redis.conf'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
```

Comment:
BTW why isn't the `redis` user owner of `/etc/redis/redis.conf` and `/etc/redis/core.conf`? (And also, do we really need a custom `core.conf`?)